AutolayoutPractice/RemindersList/Views/ReminderCell/ReminderCell.swift

```diff
@@ -44,6 +45,14 @@ class ReminderCell: UITableViewCell {
         return lbl
     }()
 
+    private let prioritySymbol: UIImageView = {
+        let img = UIImage(systemName: "exclamationmark.bubble.fill")
```

Comment:
I see you're working with a system image so no worries, but generally it's recommended to use image catalogs and image literals instead of `UIImage`. This constructor returns an optoinal image, which is not ideal if you know the image exists